### PR TITLE
fix build error

### PR DESCRIPTION
**Pull request by KeisukeShima**
## What kinds of PR

- [ ] New feature
- [x] Improve feature
- [ ] Bug fix

## What is this PR
Fix clang nightly build error.

## How to check this PR
Check workflow_dispatch job.

### CODE REVIEW

**Comment by KeisukeShima** [comment]
The branch referenced by `actions-ros-ci` has been merged and disappeared, so I'll temporarily use the master branch until a new release is created.
https://github.com/ros-tooling/action-ros-ci/pull/597

File: .github/workflows/nightly_clang_build.yml [hunk 1]
```diff
@@ -20,11 +20,12 @@ jobs:
 
       - name: Run action-ros-ci
         id: action_ros_ci_step
-        uses: ros-tooling/action-ros-ci@v0.2
+        uses: ros-tooling/action-ros-ci@add-private-repo-test
```
Comment (kenji-miyake) [inline .github/workflows/nightly_clang_build.yml]: or master?

Comment (kenji-miyake) [reply .github/workflows/nightly_clang_build.yml]: or wait for next release

**Review by kenji-miyake — DISMISSED** [review]

**Review by kenji-miyake — APPROVED** [review]

**Comment by KeisukeShima** [comment]
The build job is working, but Clang is still giving an error.
This PR is intended to resolve version issues for dependent packages, so I'll just log it here.

```
2021-05-24T07:38:33.4214213Z [6.955s] In file included from /home/runner/work/obstacle_stop_planner_refine/obstacle_stop_planner_refine/ros_ws/src/autoware.iv/common/util/autoware_utils/src/autoware_utils.cpp:15:
2021-05-24T07:38:33.4216242Z [6.955s] In file included from /home/runner/work/obstacle_stop_planner_refine/obstacle_stop_planner_refine/ros_ws/src/autoware.iv/common/util/autoware_utils/include/autoware_utils/autoware_utils.hpp:20:
2021-05-24T07:38:33.4218271Z [6.956s] In file included from /home/runner/work/obstacle_stop_planner_refine/obstacle_stop_planner_refine/ros_ws/src/autoware.iv/common/util/autoware_utils/include/autoware_utils/geometry/pose_deviation.hpp:21:
2021-05-24T07:38:33.4221091Z [6.956s] /home/runner/work/obstacle_stop_planner_refine/obstacle_stop_planner_refine/ros_ws/src/autoware.iv/common/util/autoware_utils/include/autoware_utils/math/normalization.hpp:24:18: error: constexpr function never produces a constant expression [-Winvalid-constexpr]
2021-05-24T07:38:33.4223238Z [6.957s] constexpr double normalizeDegree(const double deg, const double min_deg = -180)
2021-05-24T07:38:33.4223992Z [6.957s]                  ^
2021-05-24T07:38:33.4225865Z [6.957s] /home/runner/work/obstacle_stop_planner_refine/obstacle_stop_planner_refine/ros_ws/src/autoware.iv/common/util/autoware_utils/include/autoware_utils/math/normalization.hpp:28:22: note: non-constexpr function 'fmod' cannot be used in a constant expression
2021-05-24T07:38:33.4227513Z [6.957s]   const auto value = std::fmod(deg, 360.0);
2021-05-24T07:38:33.4228041Z [6.958s]                      ^
2021-05-24T07:38:33.4228960Z [6.958s] /usr/include/x86_64-linux-gnu/bits/mathcalls.h:168:13: note: declared here
2021-05-24T07:38:33.4229799Z [6.958s] __MATHCALL (fmod,, (_Mdouble_ __x, _Mdouble_ __y));
2021-05-24T07:38:33.4230356Z [6.959s]             ^
2021-05-24T07:38:33.4231464Z [6.959s] In file included from /home/runner/work/obstacle_stop_planner_refine/obstacle_stop_planner_refine/ros_ws/src/autoware.iv/common/util/autoware_utils/src/autoware_utils.cpp:15:
2021-05-24T07:38:33.4233420Z [6.960s] In file included from /home/runner/work/obstacle_stop_planner_refine/obstacle_stop_planner_refine/ros_ws/src/autoware.iv/common/util/autoware_utils/include/autoware_utils/autoware_utils.hpp:20:
2021-05-24T07:38:33.4235440Z [6.960s] In file included from /home/runner/work/obstacle_stop_planner_refine/obstacle_stop_planner_refine/ros_ws/src/autoware.iv/common/util/autoware_utils/include/autoware_utils/geometry/pose_deviation.hpp:21:
2021-05-24T07:38:33.4238240Z [6.960s] /home/runner/work/obstacle_stop_planner_refine/obstacle_stop_planner_refine/ros_ws/src/autoware.iv/common/util/autoware_utils/include/autoware_utils/math/normalization.hpp:36:18: error: constexpr function never produces a constant expression [-Winvalid-constexpr]
2021-05-24T07:38:33.4240359Z [6.961s] constexpr double normalizeRadian(const double rad, const double min_rad = -pi)
2021-05-24T07:38:33.4241092Z [6.961s]                  ^
2021-05-24T07:38:33.4242948Z [6.961s] /home/runner/work/obstacle_stop_planner_refine/obstacle_stop_planner_refine/ros_ws/src/autoware.iv/common/util/autoware_utils/include/autoware_utils/math/normalization.hpp:40:22: note: non-constexpr function 'fmod' cannot be used in a constant expression
2021-05-24T07:38:33.4244536Z [6.962s]   const auto value = std::fmod(rad, 2 * pi);
2021-05-24T07:38:33.4245054Z [6.962s]                      ^
2021-05-24T07:38:33.4245974Z [6.962s] /usr/include/x86_64-linux-gnu/bits/mathcalls.h:168:13: note: declared here
2021-05-24T07:38:33.4246826Z [6.963s] __MATHCALL (fmod,, (_Mdouble_ __x, _Mdouble_ __y));
2021-05-24T07:38:33.4247365Z [6.963s]             ^
2021-05-24T07:38:33.4247830Z [8.075s] 2 errors generated.
2021-05-24T07:38:33.4248836Z [8.089s] make[2]: *** [CMakeFiles/autoware_utils.dir/build.make:76: CMakeFiles/autoware_utils.dir/src/autoware_utils.cpp.o] Error 1
2021-05-24T07:38:33.4250105Z [8.089s] make[1]: *** [CMakeFiles/Makefile2:133: CMakeFiles/autoware_utils.dir/all] Error 2
2021-05-24T07:38:33.4250943Z [8.089s] make: *** [Makefile:146: all] Error 2
2021-05-24T07:38:33.4251533Z ---
2021-05-24T07:38:33.4254747Z /usr/local/lib/python3.8/dist-packages/colcon_cmake/task/cmake/build.py:172: UserWarning: colcon_core.task.check_call() has been deprecated, use colcon_core.task.run() instead
2021-05-24T07:38:33.4255987Z   return await check_call(
2021-05-24T07:38:33.4257638Z /usr/local/lib/python3.8/dist-packages/colcon_cmake/task/cmake/build.py:238: UserWarning: colcon_core.task.check_call() has been deprecated, use colcon_core.task.run() instead
2021-05-24T07:38:33.4258838Z   rc = await check_call(
2021-05-24T07:38:33.4260358Z /usr/local/lib/python3.8/dist-packages/colcon_cmake/task/cmake/build.py:342: UserWarning: colcon_core.task.check_call() has been deprecated, use colcon_core.task.run() instead
2021-05-24T07:38:33.4261542Z   return await check_call(
2021-05-24T07:38:33.4262224Z --- stderr: autoware_utils
2021-05-24T07:38:33.4263129Z CMake Deprecation Warning at /opt/ros/foxy/src/gtest_vendor/CMakeLists.txt:2 (cmake_minimum_required):
2021-05-24T07:38:33.4264260Z   Compatibility with CMake < 2.8.12 will be removed from a future version of
2021-05-24T07:38:33.4264943Z   CMake.
2021-05-24T07:38:33.4265211Z 
2021-05-24T07:38:33.4265804Z   Update the VERSION argument <min> value or use a ...<max> suffix to tell
2021-05-24T07:38:33.4266739Z   CMake that the project does not need compatibility with older versions.
2021-05-24T07:38:33.4267313Z 
2021-05-24T07:38:33.4267536Z 
2021-05-24T07:38:33.4268597Z In file included from /home/runner/work/obstacle_stop_planner_refine/obstacle_stop_planner_refine/ros_ws/src/autoware.iv/common/util/autoware_utils/src/autoware_utils.cpp:15:
2021-05-24T07:38:33.4270456Z In file included from /home/runner/work/obstacle_stop_planner_refine/obstacle_stop_planner_refine/ros_ws/src/autoware.iv/common/util/autoware_utils/include/autoware_utils/autoware_utils.hpp:20:
2021-05-24T07:38:33.4272441Z In file included from /home/runner/work/obstacle_stop_planner_refine/obstacle_stop_planner_refine/ros_ws/src/autoware.iv/common/util/autoware_utils/include/autoware_utils/geometry/pose_deviation.hpp:21:
2021-05-24T07:38:33.4275312Z /home/runner/work/obstacle_stop_planner_refine/obstacle_stop_planner_refine/ros_ws/src/autoware.iv/common/util/autoware_utils/include/autoware_utils/math/normalization.hpp:24:18: error: constexpr function never produces a constant expression [-Winvalid-constexpr]
2021-05-24T07:38:33.4277402Z constexpr double normalizeDegree(const double deg, const double min_deg = -180)
2021-05-24T07:38:33.4278135Z                  ^
2021-05-24T07:38:33.4279949Z /home/runner/work/obstacle_stop_planner_refine/obstacle_stop_planner_refine/ros_ws/src/autoware.iv/common/util/autoware_utils/include/autoware_utils/math/normalization.hpp:28:22: note: non-constexpr function 'fmod' cannot be used in a constant expression
2021-05-24T07:38:33.4281529Z   const auto value = std::fmod(deg, 360.0);
2021-05-24T07:38:33.4282021Z                      ^
2021-05-24T07:38:33.4282889Z /usr/include/x86_64-linux-gnu/bits/mathcalls.h:168:13: note: declared here
2021-05-24T07:38:33.4284475Z __MATHCALL (fmod,, (_Mdouble_ __x, _Mdouble_ __y));
2021-05-24T07:38:33.4285144Z             ^
2021-05-24T07:38:33.4286256Z In file included from /home/runner/work/obstacle_stop_planner_refine/obstacle_stop_planner_refine/ros_ws/src/autoware.iv/common/util/autoware_utils/src/autoware_utils.cpp:15:
2021-05-24T07:38:33.4288142Z In file included from /home/runner/work/obstacle_stop_planner_refine/obstacle_stop_planner_refine/ros_ws/src/autoware.iv/common/util/autoware_utils/include/autoware_utils/autoware_utils.hpp:20:
2021-05-24T07:38:33.4290130Z In file included from /home/runner/work/obstacle_stop_planner_refine/obstacle_stop_planner_refine/ros_ws/src/autoware.iv/common/util/autoware_utils/include/autoware_utils/geometry/pose_deviation.hpp:21:
2021-05-24T07:38:33.4293137Z /home/runner/work/obstacle_stop_planner_refine/obstacle_stop_planner_refine/ros_ws/src/autoware.iv/common/util/autoware_utils/include/autoware_utils/math/normalization.hpp:36:18: error: constexpr function never produces a constant expression [-Winvalid-constexpr]
2021-05-24T07:38:33.4295247Z constexpr double normalizeRadian(const double rad, const double min_rad = -pi)
2021-05-24T07:38:33.4295981Z                  ^
2021-05-24T07:38:33.4297894Z /home/runner/work/obstacle_stop_planner_refine/obstacle_stop_planner_refine/ros_ws/src/autoware.iv/common/util/autoware_utils/include/autoware_utils/math/normalization.hpp:40:22: note: non-constexpr function 'fmod' cannot be used in a constant expression
2021-05-24T07:38:33.4299503Z   const auto value = std::fmod(rad, 2 * pi);
2021-05-24T07:38:33.4300000Z                      ^
2021-05-24T07:38:33.4300871Z /usr/include/x86_64-linux-gnu/bits/mathcalls.h:168:13: note: declared here
2021-05-24T07:38:33.4301694Z __MATHCALL (fmod,, (_Mdouble_ __x, _Mdouble_ __y));
2021-05-24T07:38:33.4302215Z             ^
2021-05-24T07:38:33.4302652Z 2 errors generated.
2021-05-24T07:38:33.4303646Z make[2]: *** [CMakeFiles/autoware_utils.dir/build.make:76: CMakeFiles/autoware_utils.dir/src/autoware_utils.cpp.o] Error 1
2021-05-24T07:38:33.4304881Z make[1]: *** [CMakeFiles/Makefile2:133: CMakeFiles/autoware_utils.dir/all] Error 2
2021-05-24T07:38:33.4305731Z make: *** [Makefile:146: all] Error 2
2021-05-24T07:38:33.4306315Z ---
2021-05-24T07:38:33.4306839Z Failed   <<< autoware_utils	[ Exited with code 2 ]
```

**Comment by kenji-miyake** [comment]
Thank you!
I opened an issue. https://github.com/tier4/obstacle_stop_planner_refine/issues/32

**Pull request merged**